lib/Raven/ErrorHandler.php

```diff
@@ -155,10 +158,13 @@ public function registerExceptionHandler($call_existing_exception_handler = true
         $this->call_existing_exception_handler = $call_existing_exception_handler;
     }
 
-    public function registerErrorHandler($call_existing_error_handler = true, $error_types = -1)
+    public function registerErrorHandler($call_existing_error_handler = true, $error_types = null)
     {
+        if ($error_types == null) {
+            $error_type = error_reporting();
```

Comment:
Actually it looks like the Silent behavior might be broken with this. I'm investigating a bit more, as I'm not super familiar w/ this path.